e2e/deployers/disapp.go

```diff
@@ -115,6 +119,8 @@ func (d DiscoveredApp) WaitForResourcesDelete(ctx types.TestContext) error {
 
 func (d DiscoveredApp) IsDiscovered() bool {
 	return true
+func (d DiscoveredApp) GetType() string {
```

Comment:
Usually adding type means creating code using the type in many places, and this code is usually better move to the relevant types. Since we don't use GetType() yet maybe it is better to introduce it with the code using the type.